js/arbeidsrettet-oppfolging/HarAlleredeOppfolgingAlertstripe.js

```diff
@@ -8,7 +8,7 @@ const HarAlleredeOppfolgingAlertstripe = () => {
         <Alertstripe type="suksess" className="alleredeOppfolging blokk">
             <div className="alleredeOppfolging__innhold">
                 <Normaltekst className="alleredeOppfolging__tekst">{getLedetekst('infoside-fo.alertstripe.tekst')}</Normaltekst>
-                <a className="lenke" href="/veientilarbeid">
+                <a className="lenke" href="https://veientilarbeid.nav.no">
```

Comment:
Dette fungerer bare for prod. Ja kan få lagt til en 'node-miljøvariabel` som sier om man er i Prod eller preprod så kan man bruke den.